installation_and_upgrade/IBEX_upgrade.py

```diff
@@ -122,6 +126,8 @@ def _get_latest_existing_dir_path(release_dir, component):
 
     args = parser.parse_args()
 
+    ibex_install_utils.default_args.SERVER_ARCH = args.server_arch
+
```

Comment:
should you reassign `default_args` ? It feels it is not really a default arg then - if you need to cache what was passed on the command line maybe create `ibex_install_utils.current_args` or similarly named

Reply:
https://github.com/ISISComputingGroup/ibex_utils/pull/216